msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -122,37 +152,164 @@ The query parameters are:
 
 ### Consume a deep link from a tab
 
-When navigating to a deep link, Microsoft Teams simply navigates to the tab and provides a mechanism through the Microsoft Teams JavaScript library to retrieve the sub-entity ID if it exists.
+When navigating to a deep link, Microsoft Teams simply navigates to the tab and provides a mechanism through the Microsoft Teams JavaScript library to retrieve the sub-page ID if it exists.
 
-The [`microsoftTeams.getContext`](/javascript/api/@microsoft/teams-js#getcontext--context--context-----void-) call returns a context that includes the `subEntityId` field if the tab is navigated through a deep link.
+The [`app.getContext()`](/javascript/api/@microsoft/teams-js/app?view=msteams-client-js-latest#@microsoft-teams-js-app-getcontext&preserve-view=true) call ([`microsoftTeams.getContext`](/javascript/api/@microsoft/teams-js#getcontext--context--context-----void-) in TeamsJS v1) returns a promise that will resolve with the context that includes the `subPageId` property (or subEntityId for TeamsJS v1) if the tab is navigated through a deep link. For more information see [PageInfo interface](/javascript/api/@microsoft/teams-js/app?view=msteams-client-js-latest#@microsoft-teams-js-app-pageinfo&preserve-view=true).
 
 ## Deep linking from your tab
```

Comment:
This is the section we want to be clear in: This is a 1.x approach, in 2.x you should not be generating deep links at all and probably we should refer to the new way as "using strongly typed capabilities" and not "generating deep links"

Reply:
Tried to clarify the wording here and make it clearer that navigation with TeamsJS was the recommendation.